kytos/of.l2lsloop/main.py

```diff
@@ -124,7 +124,8 @@ def handle_message_in_packet_event(self, event):
             event_out = events.KycoMessageOutFlowMod(event.dpid, content)
             self.controller.buffers.msg_out.put(event_out)
         elif (flood_hash not in switch.flood_table or
-                (now() - switch.flood_table[flood_hash]).microseconds > FLOOD_TIMEOUT):
+                (now() - switch.flood_table[flood_hash]).microseconds >
+                POOLING_TIME):
```

Comment:
@diraol , I just saw kytos/kyco#93 with FLOOD_TIMEOUT. We have to figure out some way to avoid these dependency problems. I couldn't work without changing to POOLING_TIMEOUT because the PR was not merged.